<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>MainWindow</class>
 <widget class="QMainWindow" name="MainWindow">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>1654</width>
    <height>784</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>MainWindow</string>
  </property>
  <widget class="QWidget" name="centralWidget">
   <layout class="QGridLayout" name="gridLayout_6">
    <item row="0" column="6">
     <layout class="QVBoxLayout" name="verticalLayout">
      <item>
       <widget class="QLabel" name="label_5">
        <property name="autoFillBackground">
         <bool>true</bool>
        </property>
        <property name="text">
         <string/>
        </property>
       </widget>
      </item>
      <item>
       <widget class="QLabel" name="label_4">
        <property name="autoFillBackground">
         <bool>true</bool>
        </property>
        <property name="text">
         <string/>
        </property>
       </widget>
      </item>
     </layout>
    </item>
    <item row="3" column="2" colspan="2">
     <widget class="QGroupBox" name="groupBox_2">
      <property name="title">
       <string>Prediction options</string>
      </property>
      <layout class="QGridLayout" name="gridLayout_2">
       <item row="0" column="0">
        <widget class="QRadioButton" name="radioButton">
         <property name="text">
          <string>Average Prediction</string>
         </property>
        </widget>
       </item>
       <item row="1" column="0">
        <widget class="QRadioButton" name="radioButton_2">
         <property name="text">
          <string>Weighted Average Prediction</string>
         </property>
        </widget>
       </item>
       <item row="2" column="0">
        <widget class="QRadioButton" name="radioButton_3">
         <property name="text">
          <string>Kalman Filter Prediction</string>
         </property>
        </widget>
       </item>
      </layout>
     </widget>
    </item>
    <item row="2" column="0" rowspan="2" colspan="2">
     <widget class="QGroupBox" name="groupBox_3">
      <property name="title">
       <string>Thresholding using HSV colour-space</string>
      </property>
      <layout class="QGridLayout" name="gridLayout_4">
       <item row="0" column="0">
        <widget class="QSpinBox" name="spinBox">
         <property name="maximum">
          <number>256</number>
         </property>
        </widget>
       </item>
       <item row="0" column="1">
        <widget class="QSlider" name="vmin">
         <property name="maximum">
          <number>256</number>
         </property>
         <property name="orientation">
          <enum>Qt::Horizontal</enum>
         </property>
        </widget>
       </item>
       <item row="0" column="2">
        <widget class="QLCDNumber" name="lcdNumber"/>
       </item>
       <item row="0" column="3" rowspan="3">
        <widget class="QGroupBox" name="groupBox_4">
         <property name="title">
          <string>                            Presets</string>
         </property>
         <layout class="QGridLayout" name="gridLayout_3">
          <item row="0" column="0">
           <spacer name="horizontalSpacer">
            <property name="orientation">
             <enum>Qt::Horizontal</enum>
            </property>
            <property name="sizeHint" stdset="0">
             <size>
              <width>40</width>
              <height>20</height>
             </size>
            </property>
           </spacer>
          </item>
          <item row="0" column="1">
           <widget class="QPushButton" name="pushButton">
            <property name="text">
             <string>Red tracking</string>
            </property>
           </widget>
          </item>
          <item row="1" column="1">
           <widget class="QPushButton" name="pushButton_3">
            <property name="text">
             <string>Blue tracking</string>
            </property>
           </widget>
          </item>
          <item row="2" column="1">
           <widget class="QPushButton" name="pushButton_4">
            <property name="text">
             <string>Green tracking</string>
            </property>
           </widget>
          </item>
         </layout>
        </widget>
       </item>
       <item row="1" column="0">
        <widget class="QSpinBox" name="spinBox_2">
         <property name="maximum">
          <number>256</number>
         </property>
        </widget>
       </item>
       <item row="1" column="1">
        <widget class="QSlider" name="vmax">
         <property name="maximum">
          <number>256</number>
         </property>
         <property name="orientation">
          <enum>Qt::Horizontal</enum>
         </property>
        </widget>
       </item>
       <item row="1" column="2">
        <widget class="QLCDNumber" name="lcdNumber_2"/>
       </item>
       <item row="2" column="0">
        <widget class="QSpinBox" name="spinBox_3">
         <property name="maximum">
          <number>256</number>
         </property>
        </widget>
       </item>
       <item row="2" column="1">
        <widget class="QSlider" name="smin">
         <property name="maximum">
          <number>256</number>
         </property>
         <property name="orientation">
          <enum>Qt::Horizontal</enum>
         </property>
        </widget>
       </item>
       <item row="2" column="2">
        <widget class="QLCDNumber" name="lcdNumber_3"/>
       </item>
      </layout>
     </widget>
    </item>
    <item row="1" column="5" rowspan="3" colspan="2">
     <widget class="QCustomPlot" name="widget" native="true">
      <zorder>groupBox</zorder>
      <zorder>plainTextEdit</zorder>
      <zorder>groupBox_2</zorder>
      <zorder>plainTextEdit</zorder>
     </widget>
    </item>
    <item row="3" column="4">
     <widget class="QGroupBox" name="groupBox">
      <property name="title">
       <string>Graph options</string>
      </property>
      <layout class="QGridLayout" name="gridLayout">
       <item row="0" column="0">
        <widget class="QLabel" name="label_3">
         <property name="text">
          <string>Accuracy</string>
         </property>
        </widget>
       </item>
       <item row="0" column="1">
        <widget class="QLCDNumber" name="lcdNumber_4"/>
       </item>
       <item row="1" column="0" colspan="2">
        <widget class="QPushButton" name="enableDisableGraphBtn">
         <property name="text">
          <string>Enable graph</string>
         </property>
        </widget>
       </item>
      </layout>
     </widget>
    </item>
    <item row="0" column="0" colspan="3">
     <widget class="QLabel" name="label">
      <property name="mouseTracking">
       <bool>true</bool>
      </property>
      <property name="autoFillBackground">
       <bool>true</bool>
      </property>
      <property name="text">
       <string/>
      </property>
      <property name="scaledContents">
       <bool>false</bool>
      </property>
     </widget>
    </item>
    <item row="0" column="3" colspan="3">
     <widget class="QLabel" name="label_2">
      <property name="mouseTracking">
       <bool>false</bool>
      </property>
      <property name="autoFillBackground">
       <bool>true</bool>
      </property>
      <property name="text">
       <string/>
      </property>
      <property name="scaledContents">
       <bool>false</bool>
      </property>
     </widget>
    </item>
    <item row="1" column="1">
     <widget class="QPushButton" name="pushButton_2">
      <property name="text">
       <string>Remove selection</string>
      </property>
     </widget>
    </item>
    <item row="1" column="0">
     <widget class="QPushButton" name="pauseResumeButton">
      <property name="text">
       <string>Pause</string>
      </property>
     </widget>
    </item>
    <item row="1" column="2" rowspan="2" colspan="3">
     <widget class="QPlainTextEdit" name="plainTextEdit">
      <property name="readOnly">
       <bool>true</bool>
      </property>
     </widget>
    </item>
   </layout>
  </widget>
  <widget class="QMenuBar" name="menuBar">
   <property name="geometry">
    <rect>
     <x>0</x>
     <y>0</y>
     <width>1654</width>
     <height>21</height>
    </rect>
   </property>
  </widget>
  <widget class="QToolBar" name="mainToolBar">
   <attribute name="toolBarArea">
    <enum>TopToolBarArea</enum>
   </attribute>
   <attribute name="toolBarBreak">
    <bool>false</bool>
   </attribute>
  </widget>
  <widget class="QStatusBar" name="statusBar"/>
 </widget>
 <layoutdefault spacing="6" margin="11"/>
 <customwidgets>
  <customwidget>
   <class>QCustomPlot</class>
   <extends>QWidget</extends>
   <header>qcustomplot.h</header>
   <container>1</container>
  </customwidget>
 </customwidgets>
 <resources/>
 <connections>
  <connection>
   <sender>vmin</sender>
   <signal>valueChanged(int)</signal>
   <receiver>lcdNumber</receiver>
   <slot>display(int)</slot>
   <hints>
    <hint type="sourcelabel">
     <x>152</x>
     <y>353</y>
    </hint>
    <hint type="destinationlabel">
     <x>165</x>
     <y>347</y>
    </hint>
   </hints>
  </connection>
  <connection>
   <sender>vmax</sender>
   <signal>valueChanged(int)</signal>
   <receiver>lcdNumber_2</receiver>
   <slot>display(int)</slot>
   <hints>
    <hint type="sourcelabel">
     <x>143</x>
     <y>394</y>
    </hint>
    <hint type="destinationlabel">
     <x>183</x>
     <y>392</y>
    </hint>
   </hints>
  </connection>
  <connection>
   <sender>smin</sender>
   <signal>valueChanged(int)</signal>
   <receiver>lcdNumber_3</receiver>
   <slot>display(int)</slot>
   <hints>
    <hint type="sourcelabel">
     <x>134</x>
     <y>431</y>
    </hint>
    <hint type="destinationlabel">
     <x>163</x>
     <y>429</y>
    </hint>
   </hints>
  </connection>
  <connection>
   <sender>spinBox</sender>
   <signal>valueChanged(int)</signal>
   <receiver>vmin</receiver>
   <slot>setValue(int)</slot>
   <hints>
    <hint type="sourcelabel">
     <x>30</x>
     <y>353</y>
    </hint>
    <hint type="destinationlabel">
     <x>118</x>
     <y>353</y>
    </hint>
   </hints>
  </connection>
  <connection>
   <sender>spinBox_2</sender>
   <signal>valueChanged(int)</signal>
   <receiver>vmax</receiver>
   <slot>setValue(int)</slot>
   <hints>
    <hint type="sourcelabel">
     <x>21</x>
     <y>394</y>
    </hint>
    <hint type="destinationlabel">
     <x>90</x>
     <y>394</y>
    </hint>
   </hints>
  </connection>
  <connection>
   <sender>spinBox_3</sender>
   <signal>valueChanged(int)</signal>
   <receiver>smin</receiver>
   <slot>setValue(int)</slot>
   <hints>
    <hint type="sourcelabel">
     <x>20</x>
     <y>432</y>
    </hint>
    <hint type="destinationlabel">
     <x>87</x>
     <y>431</y>
    </hint>
   </hints>
  </connection>
  <connection>
   <sender>vmin</sender>
   <signal>valueChanged(int)</signal>
   <receiver>spinBox</receiver>
   <slot>setValue(int)</slot>
   <hints>
    <hint type="sourcelabel">
     <x>117</x>
     <y>353</y>
    </hint>
    <hint type="destinationlabel">
     <x>29</x>
     <y>353</y>
    </hint>
   </hints>
  </connection>
  <connection>
   <sender>vmax</sender>
   <signal>valueChanged(int)</signal>
   <receiver>spinBox_2</receiver>
   <slot>setValue(int)</slot>
   <hints>
    <hint type="sourcelabel">
     <x>121</x>
     <y>394</y>
    </hint>
    <hint type="destinationlabel">
     <x>23</x>
     <y>394</y>
    </hint>
   </hints>
  </connection>
  <connection>
   <sender>smin</sender>
   <signal>valueChanged(int)</signal>
   <receiver>spinBox_3</receiver>
   <slot>setValue(int)</slot>
   <hints>
    <hint type="sourcelabel">
     <x>77</x>
     <y>431</y>
    </hint>
    <hint type="destinationlabel">
     <x>20</x>
     <y>432</y>
    </hint>
   </hints>
  </connection>
  <connection>
   <sender>pushButton</sender>
   <signal>clicked()</signal>
   <receiver>MainWindow</receiver>
   <slot>setValuesAccordingToButton()</slot>
   <hints>
    <hint type="sourcelabel">
     <x>362</x>
     <y>408</y>
    </hint>
    <hint type="destinationlabel">
     <x>695</x>
     <y>697</y>
    </hint>
   </hints>
  </connection>
  <connection>
   <sender>pushButton_2</sender>
   <signal>pressed()</signal>
   <receiver>MainWindow</receiver>
   <slot>removeSelectionLogic()</slot>
   <hints>
    <hint type="sourcelabel">
     <x>298</x>
     <y>294</y>
    </hint>
    <hint type="destinationlabel">
     <x>320</x>
     <y>575</y>
    </hint>
   </hints>
  </connection>
  <connection>
   <sender>enableDisableGraphBtn</sender>
   <signal>pressed()</signal>
   <receiver>MainWindow</receiver>
   <slot>displayGraphButton()</slot>
   <hints>
    <hint type="sourcelabel">
     <x>1089</x>
     <y>744</y>
    </hint>
    <hint type="destinationlabel">
     <x>1147</x>
     <y>647</y>
    </hint>
   </hints>
  </connection>
  <connection>
   <sender>pauseResumeButton</sender>
   <signal>pressed()</signal>
   <receiver>MainWindow</receiver>
   <slot>pauseResumeButtonLogic()</slot>
   <hints>
    <hint type="sourcelabel">
     <x>71</x>
     <y>294</y>
    </hint>
    <hint type="destinationlabel">
     <x>130</x>
     <y>576</y>
    </hint>
   </hints>
  </connection>
  <connection>
   <sender>pushButton_3</sender>
   <signal>pressed()</signal>
   <receiver>MainWindow</receiver>
   <slot>setValuesForBlue()</slot>
   <hints>
    <hint type="sourcelabel">
     <x>368</x>
     <y>464</y>
    </hint>
    <hint type="destinationlabel">
     <x>463</x>
     <y>755</y>
    </hint>
   </hints>
  </connection>
  <connection>
   <sender>pushButton_4</sender>
   <signal>pressed()</signal>
   <receiver>MainWindow</receiver>
   <slot>setValuesForRed()</slot>
   <hints>
    <hint type="sourcelabel">
     <x>328</x>
     <y>533</y>
    </hint>
    <hint type="destinationlabel">
     <x>378</x>
     <y>827</y>
    </hint>
   </hints>
  </connection>
 </connections>
 <slots>
  <slot>setValuesAccordingToButton()</slot>
  <slot>removeSelectionLogic()</slot>
  <slot>displayGraphButton()</slot>
  <slot>pauseResumeButtonLogic()</slot>
  <slot>setValuesForBlue()</slot>
  <slot>setValuesForRed()</slot>
 </slots>
</ui>
